src/aap_eda/services/ruleset/activate_rulesets.py

```diff
@@ -313,7 +313,7 @@ def _restart_activation(
             logger.info(msg)
 
         activation_db_logger.write(msg)
-        enqueue_restart_task(seconds, activation.id)
+        # no further action is needed. The scheduler knows how to restart
```

Comment:
Seems like this function do nothing except logging messages. The real restart logic is handled by the scheduler, is this expected? My understanding is scheduler only do the healthy checking. If it's expected, should we remove these checkings (on_complete, on_failure, ...)?

Reply:
Yes. This function now only does proper logging. The logging messages are different on_complete or on_failure and cannot be done by scheduler because it does not know the context.

Reply:
I renamed the functions to _log_xx to make them more meaningful.